src/engine/blocks.js

```diff
@@ -270,11 +270,13 @@ class Blocks {
     /**
      * Block management: change block field values
      * @param {!object} args Blockly change event to be processed
+     * @param {?Runtime} optRuntime Optional runtime to allow changeBlock to emit actions.
      */
-    changeBlock (args) {
+    changeBlock (args, optRuntime) {
         // Validate
         if (['field', 'mutation', 'checkbox'].indexOf(args.element) === -1) return;
         const block = this._blocks[args.id];
+        let wasMonitored = block.isMonitored;
```

Comment:
Thanks @CosmicWebServices for noting the lines that fail in Travis. But you don't need to comment because we get emails whenever a build fails. GitHub also doesn't let us merge the PR if the Travis build failed.